exports.update_user = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).send({ message: error.message });
    }

}

exports.get_user = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).send({ message: error.message });
    }

}

exports.get_users = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).send({ message: error.message });
    }

}

exports.delete_user = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).send({ message: error.message });
    }

}